.github/workflows/doctests.yml

```diff
@@ -0,0 +1,80 @@
+name: Doctests
+
+on:
+  push:
+    branches:
+      - doctest*
+  repository_dispatch:
+  schedule:
+    - cron: "17 2 * * *"
+
+
+env:
+  HF_HOME: /mnt/cache
+  RUN_SLOW: yes
+  OMP_NUM_THREADS: 16
+  MKL_NUM_THREADS: 16
+  SIGOPT_API_TOKEN: ${{ secrets.SIGOPT_API_TOKEN }}
```

Comment:
Is this for https://sigopt.com/ ? Any reason we need it?

Reply:
Removed since not relevant to Diffusers :)